modules/nf-core/xengsort/index/meta.yml

```diff
@@ -0,0 +1,58 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "xengsort_index"
+description: Fast lightweight accurate xenograft sorting
+keywords:
+  - index
+  - QC
+  - reference
+  - fasta
+tools:
+  - "xengsort":
+      description: "A fast xenograft read sorter based on space-efficient k-mer hashing"
+      homepage: "https://gitlab.com/genomeinformatics/xengsort"
+      documentation: "https://gitlab.com/genomeinformatics/xengsort"
+      tool_dev_url: "https://gitlab.com/genomeinformatics/xengsort"
+      doi: "10.4230/LIPIcs.WABI.2020.4"
+      licence: ["MIT"]
+
+input:
+  - host_fasta:
+      type: file
+      description: |
+        Reference genome fasta file from host, compressed or uncompressed.
+  - graft_fasta:
+      type: file
+      description: |
+        Reference genome fasta file from graft, compressed or uncompressed.
+  - index:
+      type: string
+      description: |
+        File name prefix to store index files.
+  - nobjects:
+      type: string
+      description: |
+        Number of k-mers that will be stored in the hash table.
```

Comment:
Please add that this value uses `_` here as the thousands separator in the supplied number.

Reply:
Updated docs.